packages/assets-controllers/jest.config.js

```diff
@@ -18,7 +18,7 @@ module.exports = merge(baseConfig, {
   coverageThreshold: {
     global: {
       branches: 88.2,
-      functions: 96,
```

Comment:
I'm not 100% certain why this changed, but I checked the coverage report and all of the uncovered areas seem pre-existing.